config/prometheus/values.yaml

```diff
@@ -61,7 +61,7 @@ configReload:
   image:
     repository: jimmidyson/configmap-reload
     pullPolicy: IfNotPresent
-    tag: v0.5.0
+    tag: v0.7.1
```

Comment:
This was really just a pair of dependency bumps and a rename of the `master` branch to `main`.